pycqed/instrument_drivers/meta_instrument/qubit_objects/CCL_Transmon.py

```diff
@@ -1,7451 +1,6 @@
-                                     orders[-1]+2*dn,
-                                        orders[-1]+3*dn,
-                                        orders[-1]+4*dn)])
-        # # Checking if pulses are on 20 ns grid
-        if sweep == 'tau':
-            if not all([np.round((t*1e9)/(2*orders)) % (self.cfg_cycle_time()*1e9) == 0 for
-                      t in times]):
-                raise ValueError('timesteps must be multiples of 40e-9')
-        elif sweep == 'order':
-            if not np.round(times/2) % (self.cfg_cycle_time()*1e9) == 0:
-                raise ValueError('timesteps must be multiples of 40e-9')
-
-        # # Checking if pulses are locked to the pulse modulation
-        if sweep == 'tau':
-            if not all([np.round(t/1*1e9) % (2/self.mw_freq_mod.get()*1e9)
-                        == 0 for t in times]):
-                raise ValueError(
-                    'timesteps must be multiples of 2 modulation periods')
-
-        if prepare_for_timedomain:
-            self.prepare_for_timedomain()
-        mw_lutman = self.instr_LutMan_MW.get_instr()
-        mw_lutman.load_phase_pulses_to_AWG_lookuptable()
-        if sweep == 'tau':
-            print(times)
-            p = sqo.CPMG(times, orders, qubit_idx=self.cfg_qubit_nr(),
-                         platf_cfg=self.cfg_openql_platform_fn())
-            s = swf.OpenQL_Sweep(openql_program=p,
-                         CCL=self.instr_CC.get_instr(),
-                         parameter_name="Time", unit="s")
-        elif sweep == 'order':
-            p = sqo.CPMG_SO(times, orders, qubit_idx=self.cfg_qubit_nr(),
-                         platf_cfg=self.cfg_openql_platform_fn())
-            s = swf.OpenQL_Sweep(openql_program=p,
-                                 CCL=self.instr_CC.get_instr(),
-                                 parameter_name="Order", unit="")
-        d = self.int_avg_det
-        MC.set_sweep_function(s)
-        if sweep == 'tau':
-            MC.set_sweep_points(times)
-        elif sweep == 'order':
-            MC.set_sweep_points(orders)
-        MC.set_detector_function(d)
-        if sweep == 'tau':
-            msmt_title = 'CPMG_order_'+str(orders)+label+self.msmt_suffix
-        elif sweep == 'order':
-            msmt_title = 'CPMG_tauN_'+str(times)+label+self.msmt_suffix
-        MC.run(msmt_title)
-        if analyze:
-            # N.B. v1.5 analysis
-            if sweep == 'tau':
-                a = ma.Echo_analysis_V15(label='CPMG', auto=True, close_fig=True)
-                if update:
-                    self.T2_echo(a.fit_res.params['tau'].value)
-            elif sweep == 'order':
-                a = ma2.Single_Qubit_TimeDomainAnalysis(label='CPMG', auto=True, close_fig=True)
-
-            return a
-
-
-    def measure_spin_locking_simple(self, times=None, MC=None,
-                     analyze=True, close_fig=True, update=True,
-                     label: str = '', prepare_for_timedom
```

Comment:
Again, I feel like including the full qubit objects (and the device object like mentioned above) in the hardware-abstraction layer is just needless re-branding of these files, which will simply contribute to confusion when working with them on the laboratory. What advantages does this bring?